### PR TITLE
chore: remove puppeteer image snapshots and related story configurations

### PR DESCRIPTION
**Pull request by ozgurkececioglu**
[FX-2874]

### Description

- removal of `__image__snapshots__` folders
- removal of `waitUntilImagesLoaded` config from story files

There will be another separate PR for renaming happo project name from `picasso` to `Picasso/Cypress`. I didn't want to make that hidden behind this one.

### How to test

- you can check the file size after a fresh checkout of the repo

### Development checks

- Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- Annotate all `props` in component with documentation
- Create `examples` for component
- Ensure that deployed demo has expected results and good examples
- Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [X] Self reviewed
- Covered with tests

**Breaking change**

- codemod is created and showcased in the changeset
- test alpha package of Picasso in StaffPortal

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-2874]: https://toptal-core.atlassian.net/browse/FX-2874?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by TomasSlama — APPROVED** [review]
nice

**Review by separatio — APPROVED** [review]

File: packages/picasso/src/Quote/story/index.jsx [hunk 1]
```diff
@@ -16,7 +16,4 @@ page
   .createTabChapter('Props')
   .addComponentDocs({ component: Quote, name: 'Quote' })
 
-page.createChapter().addExample('Quote/story/Default.example.tsx', {
-  title: 'Default',
-  takeScreenshot: true,
```
Comment (LashaJini) [inline packages/picasso/src/Quote/story/index.jsx:21]: I was wondering, all other stories have set `takeScreenshot` to false, but we removed it from here :thinking:, so does it mean it has default value true? Why are we explicitly setting it to false in other stories, should we remove that parameter and clean up in the `.storybook/components/PicassoBook/Page.js` as well?

**Review by LashaJini — APPROVED** [review]
Nice. Thank you Oz :muscle:, left 1 comment only.

**Review by dmaklygin — APPROVED** [review]
good job!

**Pull request merged**